packages/create-crc-app/templates/jest.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
For RTL you should also extend the expect object

`import '@testing-library/jest-dom/extend-expect';`

in `setupFilesAfterEnv` file

Reply:
@Hyperkid123 skipped this one

Reply:
Ah yeah. :eye: